.circleci/config.yml

```diff
@@ -24,6 +26,7 @@ variables:
     name: Build and Push Service Docker Image
     command: |
       TAG=$DOCKER_TAG
+      if [[ "${REGISTRY_NAMEIMAGE}" == "" ]] ; then REGISTRY_NAMEIMAGE="buildci"; fi
```

Comment:
This, will allows to build from P.R / different username whose don't have the rights from circleCI (Pull request)

Reply:
If people fork and use CircleCI, then they will have the ability to use their own registry name.  So... this isn't needed ;)